public/scripts/search.js

```diff
@@ -0,0 +1,28 @@
+$(function() {
+    Filterometry.SearchView = Backbone.View.extend({
+        submitButton: $('#searchSubmit')[0],
```

Comment:
don't have to use [0]! technically anything you do will get applied to ALL the potential #searchSubmits.